index.html

```diff
@@ -15,13 +15,18 @@
         <header class="header">
             <nav class="nav">
                 <h1><a href="https://github.com/zeph180" ref="noreferrer">Kigozi Joseph</a></h1>
-                <i class="hamburger fas fa-bars fa-lg"></i>
+                <i id="hamburger" class="hamburger fas fa-bars fa-lg "></i>
                 <ul>
                     <li>Portfolio</li>
                     <li>About</li>
                     <li>Contact</li>
                 </ul>
             </nav>
+            <div id="mobile-menu">
+                <a class="menu-option" href="#portfolio">Portfolio</a>
+                <a class="menu-option" href="#about">About</a>
+                <a class="menu-option" href="#contact">Contact</a>
+            </div>
```

Comment:
- Good job done so far, but there are some differences between your project and the design template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. 

Your project:
-- Your mobile menu does not exactly look like what is given in the design template, please correct this 😄 
<img width="368" alt="rhwrhawrhar" src="https://user-images.githubusercontent.com/73607512/210273310-39269f1d-0b79-4bc6-8688-32bb45a87f08.PNG">

Design template:
-- 
<img width="145" alt="bqehfbqhe" src="https://user-images.githubusercontent.com/73607512/210273367-06f85c19-0a3d-42eb-a54f-f1bec53ef7cd.PNG">

Reply:
@mckent05
Thanks for this feedback let me implement that right away.